### DISCUSSION TITLE
Flyte execution tags: arbitrarily grouping executions together

### DISCUSSION THREAD

**Discussion by cosmicBboy** [flyteorg/flyte#3400] [RFC Incubator]
# Objective

Today, executions on Flyte are isolated entities, but what if I had a set of executions that I wanted to examine together? For example, I may have 20 executions of a model tuning experiment, where I want to compare the results of those executions so I can determine which is the best model?

Flyte execution tags aims to provide a primitive building block on which other features can be built, such as visualizing training metrics across multiple executions.

**Comment by kumare3** [comment]
RFC for this https://github.com/flyteorg/flyte/pull/3320

**Comment by fg91** [comment]
Closed as the RFC https://github.com/flyteorg/flyte/pull/3320 has already been accepted.
